swap_meet/item.py

```diff
@@ -1,2 +1,23 @@
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, category=None, condition=0):
```

Comment:
In regards to `category`, strings are immutable, so it's safe for us to use a string as a default value. However, it's mutable values like lists that we should be careful of as default parameters. (ex: inventory in `Vendor`)


```python
    def __init__(self, category="", condition=0.0):
```